cassandra/row_parser.pyx

```diff
@@ -33,7 +34,16 @@ def make_recv_results_rows(ColumnParser colparser):
         desc = ParseDesc(colnames, coltypes, make_deserializers(coltypes),
                          protocol_version)
         reader = BytesIOReader(f.read())
-        parsed_rows = colparser.parse_rows(reader, desc)
+        try:
+            parsed_rows = colparser.parse_rows(reader, desc)
+        except Exception as e:
+            # Use explicitly the TupleRowParser to display better error messages for column decoding failures
+            rowparser = TupleRowParser()
+            reader.buf_ptr = reader.buf
+            reader.pos = 0
+            rowcount = read_int(reader)
+            for i in range(rowcount):
+                rowparser.unpack_row(reader, desc)
```

Comment:
+1 I would be okay removing, as you suggest. I'm not aware of anyone using custom parsers outside the ones we provide.